From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Mon, 14 Sep 2026 16:51:15 +0200
Subject: [PATCH] tests: Make pass with appstream 1.2.0

Temporary fix until upstream figures out a version-independent fix.

See: https://gitlab.gnome.org/GNOME/libadwaita/-/work_items/1161
---
 tests/test-about-dialog.c | 4 ++--
 tests/test-about-window.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/test-about-dialog.c b/tests/test-about-dialog.c
index 910babb9a8fa..6106307368f0 100644
--- a/tests/test-about-dialog.c
+++ b/tests/test-about-dialog.c
@@ -19,25 +19,25 @@ test_adw_about_dialog_from_appdata (void)
   g_assert_nonnull (dialog);
 
   g_assert_cmpstr (adw_about_dialog_get_appdata_resource_path (dialog), ==, "/org/gnome/Adwaita1/Test/org.gnome.Adwaita1.Test.metainfo.xml");
-  g_assert_cmpstr (adw_about_dialog_get_release_notes (dialog), ==, "<p>Testing Build</p>\n");
+  g_assert_cmpstr (adw_about_dialog_get_release_notes (dialog), ==, "<p>Testing Build</p>");
   g_assert_cmpstr (adw_about_dialog_get_release_notes_version (dialog), ==, "1.0");
   g_assert_cmpstr (adw_about_dialog_get_version (dialog), ==, "1.0");
   g_assert_cmpstr (adw_about_dialog_get_application_icon (dialog), ==, "org.gnome.Adwaita1.Test");
   g_assert_cmpstr (adw_about_dialog_get_application_name (dialog), ==, "Adwaita Test");
   g_assert_cmpstr (adw_about_dialog_get_developer_name (dialog), ==, "The GNOME Project");
   g_assert_cmpstr (adw_about_dialog_get_issue_url (dialog), ==, "https://gitlab.gnome.org/GNOME/libadwaita/issues");
   g_assert_cmpstr (adw_about_dialog_get_support_url (dialog), ==, "https://discourse.gnome.org");
   g_assert_cmpstr (adw_about_dialog_get_website (dialog), ==, "https://gitlab.gnome.org/GNOME/libadwaita");
   g_assert_cmpuint (adw_about_dialog_get_license_type (dialog), ==, GTK_LICENSE_LGPL_2_1);
 
   g_assert_finalize_object (dialog);
 
   dialog = g_object_ref_sink (ADW_ABOUT_DIALOG (adw_about_dialog_new_from_appdata ("/org/gnome/Adwaita1/Test/org.gnome.Adwaita1.Test.metainfo.xml", "0.1")));
 
   g_assert_nonnull (dialog);
 
   g_assert_cmpstr (adw_about_dialog_get_appdata_resource_path (dialog), ==, "/org/gnome/Adwaita1/Test/org.gnome.Adwaita1.Test.metainfo.xml");
-  g_assert_cmpstr (adw_about_dialog_get_release_notes (dialog), ==, "<p>Testing Build Older</p>\n");
+  g_assert_cmpstr (adw_about_dialog_get_release_notes (dialog), ==, "<p>Testing Build Older</p>");
   g_assert_cmpstr (adw_about_dialog_get_release_notes_version (dialog), ==, "0.1");
   g_assert_cmpstr (adw_about_dialog_get_version (dialog), ==, "1.0");
 
diff --git a/tests/test-about-window.c b/tests/test-about-window.c
index ed0aa9d4dd74..cacb960e0d2c 100644
--- a/tests/test-about-window.c
+++ b/tests/test-about-window.c
@@ -17,24 +17,24 @@ test_adw_about_window_from_appdata (void)
 
   g_assert_nonnull (window);
 
-  g_assert_cmpstr (adw_about_window_get_release_notes (window), ==, "<p>Testing Build</p>\n");
+  g_assert_cmpstr (adw_about_window_get_release_notes (window), ==, "<p>Testing Build</p>");
   g_assert_cmpstr (adw_about_window_get_release_notes_version (window), ==, "1.0");
   g_assert_cmpstr (adw_about_window_get_version (window), ==, "1.0");
   g_assert_cmpstr (adw_about_window_get_application_icon (window), ==, "org.gnome.Adwaita1.Test");
   g_assert_cmpstr (adw_about_window_get_application_name (window), ==, "Adwaita Test");
   g_assert_cmpstr (adw_about_window_get_developer_name (window), ==, "The GNOME Project");
   g_assert_cmpstr (adw_about_window_get_issue_url (window), ==, "https://gitlab.gnome.org/GNOME/libadwaita/issues");
   g_assert_cmpstr (adw_about_window_get_support_url (window), ==, "https://discourse.gnome.org");
   g_assert_cmpstr (adw_about_window_get_website (window), ==, "https://gitlab.gnome.org/GNOME/libadwaita");
   g_assert_cmpuint (adw_about_window_get_license_type (window), ==, GTK_LICENSE_LGPL_2_1);
 
   g_assert_finalize_object (window);
 
   window = ADW_ABOUT_WINDOW (adw_about_window_new_from_appdata ("/org/gnome/Adwaita1/Test/org.gnome.Adwaita1.Test.metainfo.xml", "0.1"));
 
   g_assert_nonnull (window);
 
-  g_assert_cmpstr (adw_about_window_get_release_notes (window), ==, "<p>Testing Build Older</p>\n");
+  g_assert_cmpstr (adw_about_window_get_release_notes (window), ==, "<p>Testing Build Older</p>");
   g_assert_cmpstr (adw_about_window_get_release_notes_version (window), ==, "0.1");
   g_assert_cmpstr (adw_about_window_get_version (window), ==, "1.0");
 
